### ISSUE TITLE
Cap DashBoard use of Path_Base

### ISSUE THREAD

**Issue by albertopm19** [dotnetcore/CAP#901]
Thanks for your great work giving community this great framework!!!!!

Only some minor problems: Cap.Dashboard doesn't work well with Path_Base.

In the real world services are behind proxies with a basepath.  (ej:  www.domain.com/wsCap/cap.....)

I have added suport for path_base with

Add property to  public class DashboardOptions
    
        public string PathBase { get; set; }
 
In UIMiddleware.cs  replace
                htmlBuilder.Replace("%(servicePrefix)", _options.PathMatch + "/api");
with
                htmlBuilder.Replace("%(servicePrefix)", _options.PathBase+ _options.PathMatch + "/api");

With this changes in my service in startup, i can now write:

   x.UseDashboard(opt => {
                    opt.PathMatch = "/cap";
                    opt.PathBase = Configuration["PATH_BASE"];
                });

and it works !!!!

**Comment by yang-xiaodong** [comment]
Hello, thank you for your feedback, I want to know if this configuration works  in your case?

```
x.UseDashboard(opt => {
    opt.PathMatch = Configuration["PATH_BASE"]+ "/cap";   //  eg: /wsCap/cap
});
```

**Comment by albertopm19** [comment]
No , this was the first that I tried but doesn't works....

I put an example:

if Path_Base = "/wsCap"

and  opt.PathMatch = Configuration["PATH_BASE"]+ "/cap";  

then the pages of my service are in 

    www.mydomain.com/wsCap/wsCap/cap/Index.html

and the api calls are  www.mydomain.com/wsCap/cap/api

Thanks!!!

**Comment by yang-xiaodong** [comment]
Fixed in `5.1.1-preview-140603701`
